internal/cloudsql/instance.go

```diff
@@ -115,8 +115,8 @@ type RefreshCfg struct {
 }
 
 // Instance manages the information used to connect to the Cloud SQL instance by periodically calling
-// the Cloud SQL Admin API. It automatically refreshes the required information approximately 5 minutes
-// before the previous certificate expires (every 55 minutes).
+// the Cloud SQL Admin API. It automatically refreshes the required information approximately 4 minutes
+// before the previous certificate expires (every ~56 minutes).
```

Comment:
@enocom On second glance, is this sentence still valid? With the current refresh logic it isn't scheduling a refresh every ~55 minutes. It will schedule multiple refreshes throughout the error with the `d/2` logic right?